src/Paket.Core/PublicAPI.fs

```diff
@@ -384,7 +384,7 @@ type Dependencies(dependenciesFileName: string) =
             | None -> failwithf "Package %O is not installed in group %O." packageName groupName
             | Some resolvedPackage ->
                 let packageName = resolvedPackage.Name
-                let groupFolder = if groupName = Constants.MainDependencyGroup then "" else "/" + groupName.ToString()
+                let groupFolder = if groupName = Constants.MainDependencyGroup then "" else "/" + groupName.ToString().ToLower()
```

Comment:
the groupd should have a compare name - I will use that one.
ANd thanks for fixing!